src/app/myworkflows/sidebar-accordion/sidebar-accordion.component.ts

```diff
@@ -24,12 +32,51 @@ export class SidebarAccordionComponent implements OnInit {
   entryType$: Observable<EntryType>;
   EntryType = EntryType;
   public isRefreshing$: Observable<boolean>;
+  private sourceControlMap: Array<SourceControlBean> = [];
```

Comment:
I don't think you need this. You assign to it on line 46, but then you only use it on line 47... If so, I'd suggest getting rid of the property.